Documentation/ApiOverview/Logging/Quickstart/Index.rst

```diff
@@ -55,16 +55,14 @@ Log a simple message::
    $this->logger->warning('Something went awry, check your configuration!');
 
 
-Provide additional information with the log message::
-
-   $this->logger->error(
-     'This was not a good idea',
-     array(
-       'foo' => $bar,
-       'bar' => $foo,
-     )
-   );
+Provide additional context information with the log message::
+
+   $this->logger->error('Passing {value} was unwise.', [
+       'value' => $value,
+       'other_data' => $foo,
+   ]);
 
+Values in the message string that should vary based on the error (such as specifying what an invalid value was) should use placeholders, denoted by `{ }`.  Provide the value for that placeholder in the context array.
```

Comment:
Oops, missed before. See https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/GeneralConventions/CodingGuidelines.html#line-length
```suggestion
Values in the message string that should vary based on the error (such as
specifying what an invalid value was) should use placeholders, denoted by
`{ }`.  Provide the value for that placeholder in the context array.
```